docs/server/features/connectors/sinks/rabbitmq.md

```diff
@@ -0,0 +1,143 @@
+
+These configurations allow for enhanced security when communicating with RabbitMQ, protecting sensitive
+information from unauthorized access and ensuring compliance with security standards.
+
+```json
+{
+  "Authentication:Ssl:Enabled": true,
+  "Authentication:Ssl:CertPath": "/path/to/certificate",
+  "Authentication:Ssl:CertPassphrase": "*********",
+}
+```
+
+### Routing key extraction
+
+The RabbitMQ sink can extract routing keys from the record based on various sources to ensure that messages are correctly routed in RabbitMQ.
+
+This feature can be configured using the `RoutingKeyExtraction` option, to determine how routing keys are derived from the records.
+
+**Extracts the routing key from the stream ID using a regular expression**
+
+```json
+{
+  "RoutingKeyExtraction:Enabled": true,
+  "RoutingKeyExtraction:Source": "Stream",
+  "RoutingKeyExtraction:Expression": "your-regex-pattern"
+}
+```
+
+If the `Stream` source is selected, the routitng key is extracted from the stream ID based on the provided regular expression. If the regular expression matches part of the stream ID, that matched value is used as the routing key.
+
+**Extracts the routing key from the suffix of the stream ID**
+
+```json
+{
+  "RoutingKeyExtraction:Enabled": true,
+  "RoutingKeyExtraction:Source": "StreamSuffix"
+}
+```
+
+When the `StreamSuffix` source is chosen, the routing key is derived from the last segment of the stream ID, split by the '-' character. For example, if the stream ID is `order-2021-05-15`, the routing key would be `2021-05-15`.
+
+**Extracts the routing key from the record headers using a regular expression**
+
+```json
+{
+  "RoutingKeyExtraction:Enabled": true,
+  "RoutingKeyExtraction:Source": "Headers",
+  "RoutingKeyExtraction:Expression": "your-regex-pattern",
+  "RoutingKeyExtraction:HeaderKey": "your-header-key"
+}
+```
+
+If the `Headers` source is selected, the routing key is extracted from the headers of the record using the provided regular expression and header key.
+
+The connector searches through the headers, and if a match is found based on the regular expression and header key, that value is used as the routing key. If no match is found, the record's key is
+used instead.
+
+**Use the record key as the routing key**
+
+```json
+{
+  "RoutingKeyExtraction:Enabled": true,
+  "RoutingKeyExtraction:Source": "RecordKey"
+}
+```
+
+When the `RecordKey` source is chosen, the routing key is set to the value of the record key.
+
+## Settings
+
+::: note
+The RabbitMQ sink inherits a set of common settings that are used to configure the connector. The settings can be found in
+the [common settings](../settings.md) page.
+:::
+
+<!--
+The RabbitMQ sink can be configured with the following options:
```

Comment:
@YoEight Should this section about RabbitMQ options be removed, or do these still need to be added?